index.js

```diff
@@ -105,11 +113,23 @@ class PearIPC extends ReadyResource {
     this.stream.on('close', onclose)
 
     this._register()
+
+    if (this.server === null && this.id === -1) {
+      await this._beat()
+      this._heartbeat = setInterval(() => {
+        this._beat()
+      }, HEARTBEAT_INTERVAL)
```

Comment:
unref?

Reply:
discussed: we're using "global" ref/unref so makes sense to explicitly manage all refs here